sql/core/src/test/scala/org/apache/spark/sql/execution/ExecuteImmediateEndToEndSuite.scala

```diff
@@ -38,7 +38,8 @@ class ExecuteImmediateEndToEndSuite extends QueryTest with SharedSparkSession {
     }
   }
 
-  test("EXEC IMMEDIATE STACK OVERFLOW") {
+  // TODO(SPARK-49622) Re-enable `EXEC IMMEDIATE STACK OVERFLOW` test case
+  ignore("EXEC IMMEDIATE STACK OVERFLOW") {
```

Comment:
Maybe we should remove this test completely. We already have the parser unit test to verify the fix, and it doesn't seem to be necessary to run this very slow end-to-end test in the CI.

Reply:
Thank you. Ya, let's remove this.